src/locales/cs.ts

```diff
@@ -0,0 +1,797 @@
+    jarlath_of_tuam_bishop: 'Sv. Jarláta, biskupa',
+    jerome_emiliani: 'Sv. Jeronýma Emilianiho',
+    jerome_of_stridon_priest: 'Sv. Jeronýma, kněze a učitele církve',
+    joachim_he_kaizhi_martyr: 'Sv. Jáchyma He Kaizhi, mučedníka',
+    joan_of_arc_virgin_copatroness_of_france: 'Sv. Jany z Arku, panny a spolupatronky Francie',  // TODO: Or `Johanka`?
+    joaquina_of_saint_francis_of_assisi_de_vedruna_religious: 'Sv. Jáchymy od sv. Františka z Assisi de Vedruna, řeholnice',
+    john_baptist_de_la_salle_priest: 'Sv. Jana Křtitele de la Salle, kněze',
+    john_berchmans_religious: 'Sv. Jana Berchmanse, řeholníka',  // src: http://www.jesuit.cz/kalendarium/kalendarium.php?id=58&opakovani=1574722800
+    john_bosco_priest: 'Sv. Jana Boska, kněze',
+    john_cassian_priest: 'Sv. Jana Kassiána, kněze',
+    john_chrysostom_bishop: 'Sv. Jana Zlatoústého, biskupa a učitele církve',
+    john_damascene_priest: 'Sv. Jana Damašského, kněze a učitele církve',
+    john_de_britto_priest: 'Sv. Jana de Brito, kněze a mučedníka',
+    john_eudes_priest: 'Sv. Jana Eudese, kněze',
+    john_gabriel_perboyre_priest: 'Sv. Jana Gabriela Perboyre, kněze a mučedníka',
+    john_i_pope: 'Sv. Jana I., papeže a mučedníka',
+    john_jones_priest: 'Sv. Jana Jonese, kněze a mučedníka',  // src: http://www.sfr.cz/breviar_detail.aspx?a=317
+    john_leonardi_priest: 'Sv. Jana Leonardiho, kněze',
+    john_macias_religious: 'Sv. Jana Macíase, řeholníka',  // src: http://breviar.op.cz/?a=7&datum=2017-09-18&k=18SEP&chyba=1
+    john_mary_vianney_priest: 'Sv. Jana Marie Vianneye, kněze',
+    john_nepomucene_priest: 'Sv. Jana Nepomuckého, kněze a mučedníka',
+    john_nepomucene_neumann_bishop: 'Sv. Jana Nepomuckého Neumanna, biskupa',
+    john_of_avila_priest: 'Sv. Jana z Ávily, kněze a učitele církve',
+    john_of_capistrano_priest: 'Sv. Jana Kapistránského, kněze',
+    john_of_dukla_priest: 'Sv. Jana z Dukly, kněze',
+    john_of_god_duoarte_cidade_religious: 'Sv. Jana z Boha, řeholníka',
+    john_of_kanty_priest: 'Sv. Jana Kentského, kněze',
+    john_of_the_cross_priest: 'Sv. Jana od Kříže, kněze a učitele církve',
+    john_of_triora_priest: 'Sv. Jana z Triora, kněze a mučedníka',
+    john_ogilvie_priest: 'Sv. Jana Ogilvie, kněze a mučedníka',
+    john_paul_ii_pope: 'Sv. Jana Pavla II., papeže',
+    john_roberts_priest: 'Sv. Jana Robertse, kněze a mučedníka',
+    john_sarkander_priest: 'Sv. Jana Sarkandra, kněze a mučedníka',
+    john_apostle: 'Sv. Jana, apoštola a evangelisty',
+    john_xxiii_pope: 'Sv. Jana XXIII., papeže',
+    josaphat_kuntsevych_bishop: 'Sv. Josafata, biskupa a mučedníka',
+    joseph_de_anchieta_priest: 'Sv. Josefa de Anchieta, kněze',
+    joseph_bilczewski_bishop: 'Sv. Josefa Bilczewského, biskupa',
+    jose_maria_de_yermo_y_parres_priest: 'Sv. Josefa Marie de Yermo y Parres, kněze',
+    josemaria_escriva_de_balaguer_priest: 'Sv. Josemaríe Escrivá de Balaguer, kněze',
```

Comment:
I'm not too acquainted with this corner of the church, but I believe the first name is usually left uninflected and the second one is being inflected:
`Sv. Josemaría Escrivy de Balaguer, kněze`
cf. https://zpravy.proglas.cz/udalosti/opus-dei-ma-noveho-prelata-fernardo-ocariz-bra-o/
https://paloucek.wixsite.com/paloucek/about

Reply:
Your first link (zpravy.proglas.cz) contains that name in genitive as `svatého Josemaria Escrivy`, however, the other link (paloucek.wixsite.com) contains `Josemaríi Escrivu`:
- papež Jan Pavel II. monsignora _**Escrivu**_ na Svatopetrském náměstí v Římě svatořečil;
- Katolická církev slaví svátek sv. _**Josemaríi**_;
- Prvním nástupcem sv. _**Josemaríi**_ v čele Opus Dei;
-  stal se nejbližším spolupracovníkem svatého _**Josemaríi**_.

(Correct me if I’m wrong.)

Reply:
Ping, @igneus.